src/components/issues/SearchField.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect, ChangeEvent } from 'react';
+import { useRouter } from 'next/router';
+import { getQueryStringFromInput } from '@/utils/getQueryStringFromInput';
+import { getQueryStringFromUrl } from '@/utils/getQueryStringFromUrl';
+import classNames from '@/styles/issues.module.scss';
+import { handleFeatureFlag } from '@/utils/handleFeatureFlag';
+import { updateQueryStringToUrl } from '@/utils/updateQueryStringToUrl';
+
+type SearchFieldProps = {
+    onSearchTextSubmitted: (querySearchString: string) => void;
+    loading: boolean;
+};
+
+const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
+    const router = useRouter();
+    const dev = router?.query?.dev;
+    const qQueryParam = router?.query?.q;
+    const [searchText, setSearchText] = useState<string>('');
+    const onSearchTextChanged = (e: ChangeEvent<HTMLInputElement>) => {
+        setSearchText(e.target.value);
+    };
+
+    const queryParamValue = getQueryStringFromInput(searchText);
+
+    const handleOnSubmit = (e: React.FormEvent) => {
+        e.preventDefault();
+        onSearchTextSubmitted(searchText);
+        handleFeatureFlag(router?.query?.dev?.toString(), () =>
```

Comment:
`router?.query?.dev?.toString()`
The `dev` part is already available in a variable above.

Reply:
Done.